.github/workflows/stale.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '00 19 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
```

Comment:
Is there a way to configure this to _add_ a label when it unstales an issue? What I really want to avoid is

1. User files
2. We ask for info
3. They respond
4. we miss it
5. Bot stales
6. Bot closes

Because that's super annoying.

Reply:
This is something I looked for but couldn't find support for. Created https://github.com/actions/stale/issues/460 that this PR should potentially be blocked on. I have no issue removing "Needs: Author Feedback" as issues are marked un-stale though.